apps/website/src/components/notifications/Schedule.tsx

```diff
@@ -64,20 +64,24 @@ export function Schedule() {
     () => today && new Date(today.getFullYear(), today.getMonth(), 1),
     [today],
   );
-  const daysInMonth = useMemo(
+  const endOfMonth = useMemo(
     () =>
       startOfMonth &&
       new Date(
         startOfMonth.getFullYear(),
         startOfMonth.getMonth() + 1,
         0,
-      ).getDate(),
+        23,
+        59,
+        59,
```

Comment:
The logic in the API does a `<`, so do we want this to be `1, 0, 0, 0` rather than `0, 23, 59, 59`? (The `0,0,0` wouldn't be needed)

Reply:
ah yeah